### ISSUE TITLE
Create setDanceSpeedEach API

### ISSUE THREAD

**Issue by ryansloan** [code-dot-org/dance-party#248]
Used for https://levelbuilder-studio.code.org/blocks/Dancelab_setDanceSpeedEach/edit

I have removed this block from the Groups category of /s/dance puzzle 13
```
    <block type="Dancelab_setDanceSpeedEach">
      <title name="GROUP">sprites</title>
      <title name="SPEED">2</title>
    </block>
```

**Comment by ryansloan** [comment]
FYI @mrjoshida

**Comment by ryansloan** [comment]
This is fixed in #420
